ee/tabby-webserver/src/schema/mod.rs

```diff
@@ -435,13 +453,40 @@ impl Mutation {
         smtp_password: Option<String>,
         smtp_server: String,
     ) -> Result<bool> {
+        let Some(JWTPayload { is_admin: true, .. }) = &ctx.claims else {
```

Comment:
extracting this into a function like?
```
fn check_admin() -> Result<()>
```

Reply:
That was going to be the refactor I do when implementing the owner/admin logic in my next PR